src/pilot/config_dashboard.py

```diff
@@ -350,254 +293,304 @@ def prep_ceph_hosts(dashboard_node, ceph_nodes):
         node.run("sudo restorecon {}".format(Node.etc_hosts))
         os.unlink(tmp_hosts)
 
-    # configure access now
-    configure_access(ceph_nodes)
-
 
-def prep_calamari_hosts(calamari_node, ceph_nodes):
-    """Prepares the /etc/hosts file on the Calamari server
-
-    NOTE: See https://bugzilla.redhat.com/show_bug.cgi?id=1414918
-          This function implements a workaround to the BZ.
-
-    The Storage Console likes to identify nodes by their FQDN, which is
-    typically what you see when running "hostname --fqdn" on each
-    node. However, when the Storage Console imports an external Ceph cluster,
-    it uses the Calamari server (running on a Ceph monitor node, such as
-    controller-0) to acquire the FQDNs of every MON and OSD node in the
-    cluster.
+def prep_root_user(dashboard_node, ceph_nodes):
+    """ Prepares root user on the Ceph Storage Dashboard Console
+    Modifies the Ceph Storage Nodes so that Ceph Storage Dashboard
+    Console can access so that root can install dashboard.
+    """
 
-    The Calamari server uses the contents of its own /etc/hosts file to map
-    the IP address of each node to the FQDN that it reports to the Storage
-    Console, and the hosts file is created by the OSP Director. Unfortunately,
-    the Director adds an extra ".storage" subdomain to all Controllers'
-    storage network address. This causes the Calamari server to report an
-    FQDN for all monitors that don't match their actual FQDN, and that ends
-    up confusing the Storage Console.
+    home = expanduser("~")
+    root_key = Node.root_home + "/.ssh/authorized_keys"
+    tmp_key = "/tmp/authorized_keys"
+    repl_str = ".*sleep 10\" "
+    bak_date = dashboard_node.run('date +"%Y%m%d%H%M"')
+
```

Comment:
It might be helpful to add comment here indicating that it's a check for existence of file